ros/openapi/openapi.json

```
@@ -108,6 +110,7 @@
             "get": {
                 "summary": "Show the system's performance history by given inventory id",
                 "description": "Show the system's performance history by given inventory id",
+                "operationId": "getSystemsPerformanceHistory",
```

Can we make it similar to `getSystemDetails` since we are talking about single system?
```suggestion
                "operationId": "getSystemPerformanceHistory",
```